### PR TITLE
 array.at with negative indexes 

### PR DESCRIPTION
**Pull request by Graur**
#1686 

What's done:

- added block with negative indexes
- added several tests

### CODE REVIEW

**Comment by Graur** [comment]
@volodya-lombrozo Please, take a look

File: eo-runtime/src/test/java/EOorg/EOeolang/EOarrayEOatTest.java [hunk 1]
```diff
@@ -58,4 +60,32 @@ public void pushesAndGetsBack() {
             Matchers.equalTo(txt)
         );
     }
+
+    @Test
+    public void checksNegativeIndex() {
```
Comment (volodya-lombrozo) [inline eo-runtime/src/test/java/EOorg/EOeolang/EOarrayEOatTest.java:65]: @Graur We have significant duplication between all three tests. Could we remove it?

**Review by volodya-lombrozo — CHANGES_REQUESTED** [review]

File: eo-runtime/src/test/java/EOorg/EOeolang/EOarrayEOatTest.java (hunk 1 above)
Comment (Graur) [reply eo-runtime/src/test/java/EOorg/EOeolang/EOarrayEOatTest.java:64]: @volodya-lombrozo I think I got the idea. Let me fix this

Comment (Graur) [reply eo-runtime/src/test/java/EOorg/EOeolang/EOarrayEOatTest.java:64]: @volodya-lombrozo Check again please. Does it better now?

**Review by volodya-lombrozo — APPROVED** [review]
@Graur Thanks. Looks good to me.

**Comment by Graur** [comment]
@yegor256 Please, take a look

**Comment by yegor256** [comment]
@rultor merge

**Comment by Graur** [comment]
@yegor256 Can you try to merge it again, please?

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**